.azure-devops/merge.yml

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+
+trigger:
+  branches:
+    include:
+      - '*'
+  paths:
+    exclude:
+    - '**/*.md'
+
+pr:
+  branches:
+    include:
+      - '*'
+  paths:
+    exclude:
+    - '**/*.md'
+
+jobs:
+
+- job: 'Build'
+  displayName: 'Build wheel and sdist'
+  pool:
+    vmImage: 'ubuntu-latest'
+  steps:
+  - checkout: self
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '3.12'
```

Comment:
Should this match the min supported python version?

Reply:
I guess it depends if we want to use it for checkin / PR gates - or if it's just a dummy build for devops.
We already have the full gamut of python / OS running on PRs, but it probably wouldn't hurt to give this pipeline a small bit of utility :)